### ISSUE TITLE
Doesn't work on Linux OS

### ISSUE THREAD

**Issue by delfin86** [habx/nightwatchx#54]
Hey! Did you run framework on Linux OS? It doesn't work there...

**Comment by jean9696** [comment]
@delfin86 do you have any error message ? 
I forgot to mention in the ReadMe that you need these environement variables to be able to run the script
```
BROWSERSTACK_ACCESS_KEY=***
AWS_SECRET_ACCESS_KEY=***
SLACK_TOKEN=***

AWS_ACCESS_KEY_ID=***
SLACK_HOOK=***
SLACK_HOOK_ERROR_ONLY=***
AWS_ACCOUNT_ID=***
AWS_REGION=***
BROWSERSTACK_USERNAME=***
```

However, if you run `npm run  test:local` it should work.

Did you try the [example](https://github.com/habx/nightwatchx/tree/dev/example) ?

**Comment by delfin86** [comment]
@jean9696 hey!

No custom devices found
Please add environments in your config

I have devices.js but there is absent devices.json
Can you help me with that?

**Comment by jean9696** [comment]
@delfin86 I made environments config optional with just a warning here https://github.com/habx/nightwatchx/pull/59
I published the new version *v0.4.0*
However, I recommand you to add these files under a `config` folder as described [here](https://github.com/habx/nightwatchx/releases/tag/v0.4.0)
